cmd/discovery/cmd/listen.go

```diff
@@ -25,7 +25,18 @@ to quickly create a Cobra application.`,
 	Run: func(cmd *cobra.Command, args []string) {
 		fmt.Println("listen called")
 
-		discovery.StartService("9000", *id)
+		// use a fixed id for test
+		id := "snode://c03ff3c956d0a320b153a097c3d04efa488d43d7d7e05a44791492c9979ff558f9956c0a6b0c414783476f02ad8557349d35ba9373dadfa9a7a44fd88328189f@:9000"
```

Comment:
why don't you pass this id form cmdline?

Reply:
fix id for test, so I could use the same command line

Reply:
and it's for test, so I don't want pass it by cmd line

Reply:
nice.